qiskit_addon_dice_solver/dice_solver.py

```diff
@@ -355,7 +355,8 @@ def _write_input_files(
     # We don't want Dice to be noisyu for now so we hard code noio
     noio = "noio\n"
     # The number of determinants to write as output. We always want all of them.
-    dim = math.comb(norb, num_up) * math.comb(norb, num_dn)
+    # The larger integer in C++ is 2147483647
```

Comment:
```suggestion
    # Cap the number of determinants returned to the largest possible `int` value in C
```

Reply:
@caleb-johnson did you forget to change the code to use `ctypes.sizeof(ctypes.c_int)`?

Reply:
No, I decided against it. The platforms that use 64-bit int are niche. I think it makes sense just to do it how you did it originally